changed alignment for footer links (Mobile)

kpulkit29 opened the pull request:
Fixes #3033 

Changes: Center aligned the footer links for mobile views, were distorted earlier.

Demo Link: https://pr-3040-fossasia-susi-web-chat.surge.sh

Screenshots of the change: 
![Screenshot 2019-11-17 at 11 20 42 PM](https://user-images.githubusercontent.com/20151526/69011538-7c631e80-0991-11ea-9571-cb4546f967ed.png)
![Screenshot 2019-11-17 at 11 20 28 PM](https://user-images.githubusercontent.com/20151526/69011539-7cfbb500-0991-11ea-97d7-e742e1b9bb0d.png)

kpulkit29 commented:
> In this view, put 5 links in a row. This doesn't look good!

sure

kpulkit29 commented:
@akshatnitd I have made the requested changes and updated my branch.

akshatnitd commented on src/components/Footer/Footer.react.js:
```diff
@@ -116,19 +123,36 @@ const Footer = () => {
               <li>
                 <Link to="/support">Support</Link>
               </li>
+              {isMobileView() && (
```
Instead of calling `isMobileView()` multiple times, call it only once before return statement and use the value elsewhere.

kpulkit29 replied:
Resolved this comment

kpulkit29 commented:
@akshatnitd I have updated the branch and addressed the changes. Kindly review

akshatnitd commented:
> @akshatnitd I have updated the branch and addressed the changes. Kindly review

@kpulkit29 You have not addressed the changes requested. Still function call to `isMobileView` is unnecessarily called.

kpulkit29 commented:
> > @akshatnitd I have updated the branch and addressed the changes. Kindly review
> 
> @kpulkit29 You have not addressed the changes requested. Still function call to `isMobileView` is unnecessarily called.

That check has to be there as the display for mobile and desktop is totally different. Moreover you had asked me to reduce the number of isMobileView calls which I had addressed.

kpulkit29 commented:
Okay got your point would update the PR. Thanks!!

kpulkit29 commented:
@akshatnitd I have  addressed the change `isMobileView` is called only once now

akshatnitd commented:
Please squash the commits.

kpulkit29 commented:
@akshatnitd I have squashed my commits and updated the branch with master
